docs/conf.py

```diff
@@ -0,0 +1,336 @@
+        """Template HTML for embedding video into webpage."""
+        rst = f""".. video:: {video_filepath}
+        :autoplay:
+        :loop:
+        :width: 600
+        :poster: {os.path.splitext(video_filepath)[0] + ".png"}
+    """
+        return rst
+
+
+def reset_napari(gallery_conf, fname):
+    from napari.settings import get_settings
+    from qtpy.QtWidgets import QApplication
+
+    settings = get_settings()
+    settings.appearance.theme = "dark"
+
+    # Disabling `QApplication.exec_` means example scripts can call `exec_`
+    # (scripts work when run normally) without blocking example execution by
+    # sphinx-gallery. (from qtgallery)
+    QApplication.exec_ = lambda _: None
+
+
+# def napari_scraper(block, block_vars, gallery_conf):
+#     """Basic napari window scraper.
+
+#     Looks for any QtMainWindow instances and takes a screenshot of them.
+
+#     `app.processEvents()` allows Qt events to propagateo and prevents hanging.
+#     """
+#     imgpath_iter = block_vars['image_path_iterator']
+
+#     if app := napari.qt.get_app():
+#         app.processEvents()
+#     else:
+#         return ""
+
+#     img_paths = []
+#     for win, img_path in zip(
+#         reversed(napari._qt.qt_main_window._QtMainWindow._instances),
+#         imgpath_iter,
+#     ):
+#         img_paths.append(img_path)
+#         win._window.screenshot(img_path, canvas_only=False)
+
+#     napari.Viewer.close_all()
+#     app.processEvents()
+
+#     return scrapers.figure_rst(img_paths, gallery_conf['src_dir'])
+
+
+sphinx_gallery_conf = {
+    # path to your example scripts
+    "examples_dirs": "../examples",
+    "gallery_dirs": "gallery",  # path to where to save gallery generated output
+    "filename_pattern": "/*.py",
+    "ignore_pattern": "README.rst|/*_.py",
+    "default_thumb_file": Path(__file__).parent / "images" / "logo.png",
+    "plot_gallery": "'True'",  # https://github.com/sphinx-gallery/sphinx-gallery/pull/304/files
+    "download_all_examples": False,
+    "only_warn_on_example_error": False,
+    "abort_on_example_error": True,
+    "image_scrapers": (
+        "matplotlib",
+        VideoScraper(),
+    ),
+    "reset_modules": (reset_napari,),
+    "reference_url": {"napari": None},
+    "within_subsection_order": ExampleTitleSortKey,
+}
+
+
+def setup(app):
+    """Set up docs build.
+
+    * Ignores .ipynb files to prevent sphinx from complaining about multiple
+      files found for document when generating the gallery
+    * Rewrites github anchors to be comparable
+    * Adds google calendar api key to meetings schedule page
+
+    """
+    app.registry.source_suffix.pop(".ipynb", None)
+
+
+# linkcheck_anchors_ignore = [r'^!', r'L\d+-L\d+', r'r\d+', r'issuecomment-\d+']
+
+# linkcheck_ignore = [
+#     'https://napari.zulipchat.com/',
+#     '../_tags',
+#     'https://en.wikipedia.org/wiki/Napari#/media/File:Tabuaeran_Kiribati.jpg',
+# ]
+
+
+# def rewrite_github_anchor(app, uri: str):
```

Comment:
I don't love that there are big chunks of commented code in this file (linkcheck_ignore, rewrite_github_anchor, etc.). I know you just copy-pasted the file contents from the napari/docs repo, but what's happening here? Do we 
1. need to remove these lines entirely,
2. or does there need to be an issue opened about what problem needs to be fixed here?

Reply:
Sorry I wasn't clear - these are the scrapers from napari, and I left them here for reference. I don't know if they would work for animations, and I would need help to figure that out. Since you said you had done this before, I was hoping for a suggestion or two 😅